process-instance-migration/src/main/java/org/camunda/community/migration/processInstance/dto/client/HistoricActivityInstanceDto.java

```diff
@@ -0,0 +1,190 @@
+package org.camunda.community.migration.processInstance.dto.client;
+
+import java.time.ZonedDateTime;
+import java.util.ArrayList;
+
+public class HistoricActivityInstanceDto {
```

Comment:
What about reusing the camunda-7-rest client here?

Reply:
This would be an overkill as I only need some dtos and endpoints. So, I decided aginst using a generated client

Reply:
But it would free you from maintaining a bunch of code here and validate the usability of the external library.

Side-topic: you integrated an external library to write a CSV file.